lib/components/DonateForm.vue

```diff
@@ -1,58 +1,113 @@
+      />
+      <!-- eslint-enable -->
     </slot>
 
     <div class="donate-form__payment mb-4 text-center">
-      <form action="//checkout.fundjournalism.org/memberform" method="get" target="_blank" class="donate-form__payment__form bg-light p-4">
+      <form
+        action="//checkout.fundjournalism.org/memberform"
+        method="get"
+        target="_blank"
+        class="donate-form__payment__form bg-light p-4"
+      >
         <div class="donate-form__payment__levels row">
-          <div @click="selectLevel('conversation')" class="col donate-form__payment__level" :class="{ active: level == 'conversation' }">
-              <h3 class="donate-form__payment__heading text-uppercase font-weight-bold text-primary h5">
-                {{ $t('donate-form.benefits.impacts.conversation.heading') }}
-              </h3>
-              <div class="Article">
-                  <div>
-                    <p class="donate-form__payment__highlight text-icij font-weight-bold" v-html="$t('donate-form.benefits.impacts.conversation.highlight')"></p>
-                  </div>
+          <div
+            class="col donate-form__payment__level"
+            :class="{ active: level == 'conversation' }"
+            @click="selectLevel('conversation')"
+          >
+            <h3 class="donate-form__payment__heading text-uppercase font-weight-bold text-primary h5">
+              {{ $t('donate-form.benefits.impacts.conversation.heading') }}
+            </h3>
+            <div class="Article">
+              <div>
+                <!-- eslint-disable vue/no-v-html -->
+                <p
+                  class="donate-form__payment__highlight text-icij font-weight-bold"
+                  v-html="$t('donate-form.benefits.impacts.conversation.highlight')"
+                />
+                <!-- eslint-enable -->
               </div>
+            </div>
           </div>
-          <div @click="selectLevel('rules')" class="col donate-form__payment__level" :class="{ active: level == 'rules' }">
-              <h3 class="donate-form__payment__heading text-uppercase font-weight-bold text-primary h5">
-                {{ $t('donate-form.benefits.impacts.rules.heading') }}
-              </h3>
-              <div class="Article">
-                  <div>
-                      <p class="donate-form__payment__highlight text-icij font-weight-bold" v-html="$t('donate-form.benefits.impacts.rules.highlight')"></p>
-                  </div>
+          <div
+            class="col donate-form__payment__level"
+            :class="{ active: level == 'rules' }"
+            @click="selectLevel('rules')"
+          >
+            <h3 class="donate-form__payment__heading text-uppercase font-weight-bold text-primary h5">
+              {{ $t('donate-form.benefits.impacts.rules.heading') }}
+            </h3>
+            <div class="Article">
+              <div>  
+                <!-- eslint-disable vue/no-v-html -->
```

Comment:
Maybe adding this rule could help: 'vue/no-v-html': 'off'

Reply:
In fact it was useful for the SignupForm component (output printed out from an HTTP request to Mailchimp) so I rather keep it.

Reply:
Ok :+1: nice job!